app/utils/helpers.ts

```diff
@@ -82,6 +82,10 @@ export function isEmail(email: string): boolean {
     return (/^[^ ,@]+@[^ ,@]+$/).test(email);
 }
 
+export function identity<T>(arg: T): T {
```

Comment:
where is this being used?

Reply:
I had added this to replace `safeParseJSON`, but it looks like that we `JSON.stringify` the `value` when we save to the `System` table, so ATM we need `safeParseJSON` and cannot replace with `identity`